### PR TITLE
Inference

### PR DESCRIPTION
**Pull request by soumik12345**

### CODE REVIEW

File: multi_label_trainer_tfidf.ipynb [hunk 1]
```diff
@@ -63,9 +63,10 @@
         "from sklearn.model_selection import train_test_split\n",
```
Comment (sayakpaul) [inline multi_label_trainer_tfidf.ipynb:63]: <p>Before this, it might be a good idea to add one/two lines on the training progress plot. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (sayakpaul) [inline multi_label_trainer_tfidf.ipynb:63]: <pre class="ql-syntax" spellcheck="false">Line #7.        predicted_proba = shallow_mlp_model(text_batch)[0]
</pre><p> </p><p>If we directly call <code>.predict()</code>  we don't need to separately call <code>numpy()</code>  and won't have to worry about if our model is running in inference mode. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (sayakpaul) [inline multi_label_trainer_tfidf.ipynb:63]: <pre class="ql-syntax" spellcheck="false">Line #10.        print(f"Label(s): {', '.join([label for label in top_3_labels])}")
</pre><p> </p><p>"Predicted label(s)..." might be a better keyword.</p><p> </p><p>Also, at least add a single comment to explain what's going on in the prediction decoding. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (sayakpaul) [inline multi_label_trainer_tfidf.ipynb:63]: <p>Seems like a chore and can be deleted? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (sayakpaul) [reply multi_label_trainer_tfidf.ipynb:63]: <p>Now, we can add suggestions and other observations. Something like:</p><p> </p><p>The prediction results are not that great but not below the par for a simple model like ours. We can improve this performance with models that consider word order like LSTM or even those that use Transformers.</p><p> </p><p>It's just a hint, feel free to modify in your own words.</p><p> </p><p>Also, add at least a comment to explain the prediction decoding part. </p>

Comment (sayakpaul) [reply multi_label_trainer_tfidf.ipynb:63]: @soumik12345 only the above comment and then we can merge.

Comment (soumik12345) [reply multi_label_trainer_tfidf.ipynb:63]: @sayakpaul updated

**Review by sayakpaul — APPROVED** [review]
